stories/kitchen-sink.stories.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import KitchenSink from '../app/components/kitchen-sink';
+import common from './common'
+
+const testType = {
```

Comment:
seems like testType isn't being used here (and wasn't being used in the original kitchen-sink).  Can you delete it and verify things still works.